Changelog.md

```diff
@@ -12,7 +12,7 @@
 
 - Adding static result allocation by @idavis in https://github.com/qir-alliance/pyqir/pull/103
 - Bumping version to `0.4.0a1` by @idavis in https://github.com/qir-alliance/pyqir/pull/105
-- Changing default value of `use_static_result_alloc` to `True` by @idavis in https://github.com/qir-alliance/pyqir/pull/106
+- Changing default value of use_static_result_alloc to True by @idavis in https://github.com/qir-alliance/pyqir/pull/106
```

Comment:
This seems like an odd update to be in this PR... was this needed for something?

Reply:
@swernli  it is a formatting change that was inadvertently added, I'll reset it. It is the same content.